src/common/GasService.sol

```diff
@@ -12,7 +12,7 @@ contract GasService is IGasService {
     using MessageLib for *;
 
     /// @dev Takes into account Adapter + Gateway processing + some mismatch happened regarding the input values
-    uint128 public constant BASE_COST = 200_000;
+    uint128 public constant BASE_COST = 150_000;
```

Comment:
All values have increased by about `~50K` because now the measurement is taken into account `Gateway` + `MessageProcessor`. That's why I've removed that from the offset too.

Reply:
Yes, probably we can cut more (not sure how more)

The problem is that we have no way to estimate the computation done in the `Gateway` and `Multiadapter` when a batch is received. We can receive long messages that require more computation in the Gateway as a whole. Events will store more bytes, etc...

But maybe we can reduce a bit more yeah, `100000` is ok?